### PR TITLE
Try: Fix theme preview link

### PR DESCRIPTION
**Pull request by juanfra**
**Description**

Fixing the link problems reported [here](https://wordpress.slack.com/archives/C04EWKGDJ0K/p1728462111506899). This PR includes a few improvements:

- I'm updating the blueprint to fix the deprecation issues. 
- Adding the ability to use parameters for WP and PHP versions, so that themes that need particular versions can configure the action.
- Fixing the branch sanitization so that the preview is not broken due to mismatches with branch name vs. theme directory name.
- Bonus: Adding note markup for the note, so that it looks nicer.

---

You can see the changes in action in the [following PR](https://github.com/juanfra/twentytwentyfive/pull/2).

---

**Screencast**

https://github.com/user-attachments/assets/096d8b99-fbcb-4d9c-9b12-31cdfa858fd4

### CODE REVIEW

File: src/create-preview-links.ts [hunk 1]
```diff
@@ -67,9 +71,22 @@ function createBlueprint(
 	/* If themeDir is not provided, we assume that the action is running in a single theme workflow and the theme folder name will be the theme slug + the branch name.
 	 * If themeDir is provided, we assume that the action is running in a multi theme workflow and the theme folder name will be the theme slug.
 	 */
-	const themeFolderName = !themeDir ? `${themeSlug}-${branch}` : themeSlug;
+	const sanitizedBranch = branch.replace(/[\/\\:*?"<>|]/g, '-');
+	const themeFolderName = !themeDir
+		? `${themeSlug}-${sanitizedBranch}`
+		: themeSlug;
 	debug(`Theme folder name: ${themeFolderName}`);
```
Comment (bgrgicak) [inline src/create-preview-links.ts:78]:
```suggestion
```

This code doesn't seem to be used.

**Comment by bgrgicak** [comment]
Great work @juanfra! I also took a [look at your workflow](https://github.com/juanfra/twentytwentyfive/blob/trunk/.github/workflows/preview.yml) and everything looks good.

**Review by bgrgicak — APPROVED** [review]

**Comment by bgrgicak** [comment]
I removed some unused code ([8ac4a97](https://github.com/WordPress/action-wp-playground-pr-preview/pull/39/commits/8ac4a97ae9b73620e3cded64236e5f0c3ca70d46)), hope that you don't mind.

**Comment by juanfra** [comment]
Excellent, thanks!

**Pull request merged**